### PR TITLE
UBERF-4451: Fixed how resolved default location is applied on initial routing

### PR DESCRIPTION
**Pull request by anotherpit**
# Contribution checklist

## Brief description

If you start the app on the issue page (e.g. `/workbench/SPACE/tracker/ISSUE-42`), and then close the issue panel, you'll get to the list of all issues of the project of the initial issue (previously you was getting an empty page)

## Checklist

* [ ] - UI test added to added/changed functionality?
* [ ] - Screenshot is added to PR if applicable ?
* [x] - Does a local formatting is applied (rush format)
* [x] - Does a local svele-check is applied (rush svelte check)
* [ ] - Does a local UI tests are executed [UI Testing](../tests/readme.md)
* [x] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [ ] - Does the translations are up to date?
* [ ] - Does it well tested?
* [x] - Tested for Chrome.
* [ ] - Tested for Safari.
* [ ] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [ ] - Rebase your branch onto master and upstream branch
* [ ] - Is there any redundant or duplicate code?
* [ ] - Are required links are linked to PR?
* [ ] - Does new code is well documented ?

## Related issues

A list of closed updated issues

## Contributor requirements

* [x] - Sign-off is provided. [DCO](https://github.com/apps/dco)
* [x] - GPG Signature is provided. [GPG](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits)

### CODE REVIEW

File: plugins/tracker-resources/src/components/issues/edit/EditIssue.svelte [hunk 1]
```diff
@@ -87,7 +87,7 @@
 
   $: _id &&
     _class &&
-    queryClient.query(
+    queryClient.query<Issue>(
```
Comment (anotherpit) [inline plugins/tracker-resources/src/components/issues/edit/EditIssue.svelte:90]: Pardon for unrelated changes, made them while trying to understand how the code works, and minor TS warnings were distracting me

**Review by BykhovDenis — APPROVED** [review]

File: plugins/tracker-resources/src/components/issues/edit/EditIssue.svelte (hunk 1 above)
Comment (haiodo) [reply plugins/tracker-resources/src/components/issues/edit/EditIssue.svelte:90]: It s fine to clean warnings.

**Review by haiodo — APPROVED** [review]

File: plugins/tracker-resources/src/components/issues/edit/EditIssue.svelte (hunk 1 above)
Comment (lexiv0re) [reply plugins/tracker-resources/src/components/issues/edit/EditIssue.svelte:90]: @anotherpit just curious, did you see this warning during transpilation? I previously saw them reported as errors but only in IDEA because of their TS service was incapable of properly parsing the construction. It was all good during the build and in VSCode w/o specifying the type.
See VSCode derives the type from the first parameter:
image

Comment (anotherpit) [reply plugins/tracker-resources/src/components/issues/edit/EditIssue.svelte:90]: @lexiv0re As far as I remember there were no related warnings during transpilation. And yes, I do use IDE from JetBrains, so probably you are right, and it's only an issue with their language service, or more specifically with processing TS inside Svelte components
